gnat2goto/driver/tree_walk.adb

```diff
@@ -3715,8 +3715,11 @@ package body Tree_Walk is
             return Do_Modular_Type_Definition (N);
          when N_Floating_Point_Definition =>
             return Do_Floating_Point_Definition (N);
+         when N_Access_To_Object_Definition =>
```

Comment:
The change makes sense, but it's a bit surprising because neither commit message or body mention anything about access types.

Reply:
Added.